bitnami/airflow/Chart.yaml

```diff
@@ -32,4 +32,4 @@ name: airflow
 sources:
   - https://github.com/bitnami/bitnami-docker-airflow
   - https://airflow.apache.org/
-version: 11.3.0
+version: 11.3.1
```

Comment:
Could you bump the minor version? It is adding a non-breaking feature

Reply:
ok I have bumped the minor version